package/libpiksi/libpiksi/src/util.c

```diff
@@ -924,3 +925,11 @@ void print_trace(const char *assert_str, const char *file, const char *func, int
 
 #undef ASSERT_FAIL_MSG
 }
+
+void nanosleep_autoresume(long s, long ns)
```

Comment:
Does it bring any value to add a return value indicating number of interrupts?

Reply:
Maybe? Not sure what you'd use it for, but probably better than returning nothing?